src/styles/card.css

```diff
@@ -0,0 +1,150 @@
+.body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: inline-flex;
+  height: 406px;
+  width: 198px;
+  border: 2px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+/* #region Image */
+
+.card__item {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.card__image-item {
+  padding-top: 32px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+/* #endregion */
+
+/* #region TextInfo */
+
+.card__iformation {
+  height: 36px;
+  font-family: "Roboto", sans-serif;
```

Comment:
`font-family` is redudant here. you already set it. remove from all places except body